### PR TITLE
Add cmdlet reference for ADR Deployment creation

### PR DESCRIPTION
**Pull request by kylecweeks**
The documentation should include command line option reference.

Document also reviewed. 

@aczechowski

### CODE REVIEW

**Comment by PRMerger18** [comment]
@kylecweeks : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @kylecweeks for the contribution. Sorry for the delay, I'll try to review this by end of week.

File: sccm/sum/deploy-use/automatically-deploy-software-updates.md [hunk 1]
```diff
@@ -229,10 +229,10 @@ After you create an ADR, add additional deployments to the rule. This action hel
      - Alerts
      - Download Settings  
 
+Deployments can also be added programmatically using powerShell cmdelets. See [New-CMSoftwareUpdateDeployment](https://docs.microsoft.com/en-us/powershell/module/configurationmanager/new-cmsoftwareupdatedeployment) for a complete description of using this method.
```
Comment (aczechowski) [inline sccm/sum/deploy-use/automatically-deploy-software-updates.md]: Some minor issues to fix before publishing:
1. Change `powerShell` to `Windows PowerShell`
2. Change `cmdelets` to `cmdlets`
3. Remove `en-us/` from the URL to make it language neutral
4. Restructure the second sentence using the standard style: `For a complete description of using this method, see [New-CMSoftwareUpdateDeployment](https://docs.microsoft.com/powershell/module/configurationmanager/new-cmsoftwareupdatedeployment).`

**Comment by PRMerger9** [comment]
@kylecweeks : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *could* be today (Thursday, 18 October) around 11:00 Pacific time. No further action needed from you.

**Comment by PRMerger9** [comment]
@GitHubber17 : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Pull request merged**